doc/lsp.txt

```diff
@@ -1367,6 +1367,29 @@ which servers should be used for a given action.  The following actions are
 supported >
 
 #{
+	documentFormatting: { lspserver -> lspserver.isDocumentFormattingProvider }
 }
 <
+
+For example, if you want to use the efm-langserver for formatting, but the
+typescript-language-server for everything else:
+
+	vim9script
+
+	g:LspAddServer([
+		{
+			filetype: ['javascript', 'typescript'],
+			path: '/usr/local/bin/typescript-language-server',
+			args: ['--stdio']
+		},
+		{
+			filetype: ['efm-langserver'],
+			path: '/usr/local/bin/efm-langserver',
+			args: [],
+			customProviding: {
+				documentFormatting: (lspserver) => true
```

Comment:
Maybe this should simply be booleans instead of Funcrefs?

Reply:
Yes.  That will make it simple from a user perspective.  Users may not be aware of the Vim funcref syntax and it will look complicated.

Reply:
What do you think about using the name "features" instead of "customProviding"?

Reply:
> What do you think about using the name "features" instead of "customProviding"?

Certainly better than "customProviding". I'll change it and then we can think a bit more about it.